package.json

```diff
@@ -9,5 +9,9 @@
     "client": "cd client/ && npm start",
     "server": "cd server/ && npm run dev",
     "start": "npm run client && npm run server"
+  },
+  "dependencies": {
```

Comment:
remove these depencies. THis package.json of the root folder only stores scripts that run the program.